Add hbase logs (#713)
DataDog/integrations-extras: hbase_master/README.md

2. [Restart the Agent][9]

### Log collection

> _Available for Agent versions >6.0_

Reviewer: ```suggestion
```

We don't need this anymore 😄

1. Collecting logs is disabled by default in the Datadog Agent, you need to enable it in `datadog.yaml`:

   ```yaml
   logs_enabled: true
   ```

> 2. Add this configuration block to your `hbase_master.d/conf.yaml` file to start collecting your Active Directory Logs:

Reviewer: ```suggestion
2. Add this configuration block to your `hbase_master.d/conf.yaml` file to start collecting your Hbase_master Logs:
```

   ```yaml
   logs:
     - type: file
       path: /path/to/my/directory/file.log
       source: hbase

>        service: "<MY_SERVICE>"

Reviewer: ```suggestion
```

we don't need this here as we want customers to start using the service option in the init_config

   ```

>    Change the `path` and `MY_SERVICE` parameter values and configure them for your environment.

Reviewer: ```suggestion
   Change the `path` parameter value and configure it for your environment.
```

   See the [sample hbase_master.d/conf.yaml][10] for all available configuration options.

> 3. [Restart the Agent][11].

Reviewer: ```suggestion
3. [Restart the Agent][9].
```

### Validation

> [Run the Agent's `status` subcommand][12] and look for `hbase_master` under the Checks section.

Reviewer: ```suggestion
[Run the Agent's `status` subcommand][10] and look for `hbase_master` under the Checks section.
```

## Data Collected

### Metrics

> See [metadata.csv][13] for a list of metrics provided by this check.

Reviewer: ```suggestion
See [metadata.csv][11] for a list of metrics provided by this check.
```

## Troubleshooting

> Need help? Contact [Datadog support][14].

Reviewer: ```suggestion
Need help? Contact [Datadog support][12].
```

Author: And this is how I finally understood how reference-style links work
